Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/08-RecordMeeting/RecordMeetingFeature-02-code-0002.swift

```diff
@@ -28,15 +28,3 @@ struct AppView: View {
     }
   }
 }
-
-#Preview {
-  AppView(
-    store: Store(
-      initialState: App.State(
-        syncUpsList: SyncUpsList.State()
-      )
-    ) {
-      App()
-    }
-  )
-}
```

Comment:
📓 I removed this preview because it brings no special changes in the tutorial and was creating a confusing diff.